books/RayTracingInOneWeekend.html

```diff
@@ -3671,13 +3669,13 @@
   ### IEEE
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     “Ray Tracing in One Weekend.” raytracing.github.io/books/RayTracingInOneWeekend.html
-    (accessed MMM. DD, YYYY)
+    (accessed MMM. DD, 2023)
```

Comment:
I'm not so sure the `accessed` tag should be changed.

Aren't you supposed to clarify this as the reader when you make a citation?

Reply:
Will fix.